### DISCUSSION TITLE
Adding people search option for sponsorship with a one-time or monthly payment

### DISCUSSION THREAD

**Discussion by olekstomek** [community/community#11850] [Other Feature Feedback, Questions, & Ideas]
It could be nice to be able to find someone I can sponsor with a one-time fee. 
![image](https://user-images.githubusercontent.com/26818304/154950698-6d5d5f0b-d92b-4986-91a9-1f5157e3edee.png)
![image](https://user-images.githubusercontent.com/26818304/154951638-c11ee546-e0bb-40a3-b1ca-2012f29ae26a.png)

Some people only provide cyclical sponsorship, which does not always suit people.
![image](https://user-images.githubusercontent.com/26818304/154951507-3e5b3f3f-76ce-4af3-a2cb-1a05e6f59eaa.png)

**Comment by jlord** [comment]
We're working on this 🎉 🔜
